vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -22,12 +22,12 @@ module InlineParameterNameHints =
 
     let isValidForHint (symbol: FSharpMemberOrFunctionOrValue) =
         // is there a better way?
-        let isNotAnOperator = 
+        let isNotBuiltInOperator = 
```

Comment:
What's the point of separating built-in and non-builtin operators? Won't it be easier just to have a function for detecting both?